### PR TITLE
Added documentation about POS-integrado for Java.

### PR DESCRIPTION
**Pull request by floydnunez**
I added Java snippets to the POS Integrado documentation that's shown in:

https://transbankdevelopers.cl/documentacion/posintegrado#primeros-pasos

### CODE REVIEW

File: documentacion/posintegrado/README.md [hunk 1]
```diff
@@ -217,14 +264,20 @@ char *lastSaleResponse = last_sale();
 }
 ```
 
-El resultado de la transacción última venta devuelve los mismos datos que una venta normal y se entrega en forma de un objeto `LastSaleResponse` o un `char*` en el caso de la librería C. Si ocurre algún error al ejecutar la acción en el POS se lanzará una excepción del tipo `TransbankLastSaleException`.
+```Java
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```java
```

File: documentacion/posintegrado/README.md [hunk 2]
```diff
@@ -217,14 +264,20 @@ char *lastSaleResponse = last_sale();
...
+
+El resultado de la transacción última venta devuelve los mismos datos que una venta normal y se entrega en forma de un objeto `LastSaleResponse` o un `char*` en el caso de la librería C, o un objeto `SaleResponse` en el caso de Java. Si ocurre algún error al ejecutar la acción en el POS se lanzará una excepción del tipo `TransbankLastSaleException` en .NET o `TransbankException` en Java.
 
 ```json
 "Function": 260
 "Response": "Aprobado"
 "Commerce Code": 550062700310
 "Terminal Id": "ABC1234C"
-"Ticket": "AB123"
+"Ticket": "00086"
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
"Ticket": "ABC086"
```

File: documentacion/posintegrado/README.md [hunk 3]
```diff
@@ -270,6 +323,14 @@ RefundResp response = POS.Instance.Refund(21);
 RefundResponse response = refund(21);
 ```
 
+```Java
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```java
```

File: documentacion/posintegrado/README.md [hunk 4]
```diff
@@ -312,6 +374,14 @@ BaseResponse response = register_close();
 }
 ```
 
+```Jav
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```java
```

File: documentacion/posintegrado/README.md [hunk 5]
```diff
@@ -387,6 +462,14 @@ char *response = sales_detail(print_on_pos);
 }
 ```
 
+```Java
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```java
```

File: documentacion/posintegrado/README.md [hunk 6]
```diff
@@ -351,6 +420,12 @@ TotalsCResponse response = get_totals();
...
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```java
```

File: documentacion/posintegrado/README.md [hunk 7]
```diff
@@ -459,7 +542,15 @@ BaseResponse response = load_keys();
 }
 ```
 
-El resultado de la carga de llaves entrega en la forma de un objeto `LoadKeysResponse` o una estructura `BaseResponse` en el caso de la librería C. Si ocurre algún error al ejecutar la acción en el POS se lanzará una excepción del tipo `TransbankLoadKeysException`.
+```Jav
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```java
```

File: documentacion/posintegrado/README.md [hunk 8]
```diff
@@ -495,6 +586,14 @@ if (retval == TBK_OK){
...
 ```
 
+```Java
+import cl.transbank.pos.POS;
+//...
+boolean pollResult = POS.getInstance().poll();
+```
+
+
+
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```

File: documentacion/posintegrado/README.md [hunk 9]
```diff
@@ -517,10 +616,19 @@ if (retval == TBK_OK){
...
 ```
 
+```Java
+import cl.transbank.pos.POS;
+//...
+boolean normal = POS.getInstance().setNormalMode();
+```
+
+
+
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```

File: documentacion/posintegrado/README.md [hunk 10]
```diff
@@ -517,10 +616,19 @@ if (retval == TBK_OK){
 }
 ```
 
+```Java
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```java
```

File: documentacion/posintegrado/README.md [hunk 11]
```diff
@@ -495,6 +586,14 @@ if (retval == TBK_OK){
...
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```java
```

File: documentacion/posintegrado/README.md [hunk 12]
```diff
@@ -517,10 +616,19 @@ if (retval == TBK_OK){
...
+```
+
+
+
 <aside class="notice">
-Si el POS Integrado se cambia a modo normal, debe ser configurado nuevamente en modo Integrado siguiendo las instrucciones disponibles descritas en [Cambio a POS Integrado](referencia/posintegrado#cambio-modalidad-pos-integrado)
+Si el POS Integrado se cambia a modo normal, debe ser configurado nuevamente en el POS para regresar al modo Integrado, siguiendo las instrucciones disponibles descritas en [Cambio a POS Integrado](referencia/posintegrado#cambio-modalidad-pos-integrado), pues no es posible realizar esta configuración a través del SDK.
 </aside>
 
+
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```

File: documentacion/posintegrado/README.md [hunk 13]
```diff
@@ -387,6 +462,14 @@ char *response = sales_detail(print_on_pos);
...
 ```
 
+```Java
+import cl.transbank.pos.POS;
+//...
+List<DetailResponse> ldr = POS.getInstance().details(false);
+```
+
+
+
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```

File: documentacion/posintegrado/README.md [hunk 14]
```diff
@@ -312,6 +374,14 @@ BaseResponse response = register_close();
...
 ```
 
+```Jav
+import cl.transbank.pos.POS;
+//...
+CloseResponse cr = POS.getInstance().close();
+```
+
+
+
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```

File: documentacion/posintegrado/README.md [hunk 15]
```diff
@@ -288,9 +349,10 @@ Como respuesta el **POS** enviará un código de aprobación, acompañado de un
 Este comando es gatillado por la caja y no recibe parámetros. El POS ejecuta la transacción de cierre contra el Autorizador (no se contempla Batch Upload). Como respuesta el POS Integrado enviará un aprobado o rechazado. (Puedes ver la tabla de respuestas en este [link](/referencia/posintegrado#tabla-de-respuestas))
 
 <aside class="success">
-Esta transacción también realiza el cambió de llaves.
+Esta transacción también realiza el cambio de llaves.
 </aside>
 
+
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```

File: documentacion/posintegrado/README.md [hunk 16]
```diff
@@ -270,6 +323,14 @@ RefundResp response = POS.Instance.Refund(21);
...
 ```
 
+```Java
+import cl.transbank.pos.POS;
+//...
+RefundResponse response = POS.getInstance().refund(21);
+```
+
+
+
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```

File: documentacion/posintegrado/README.md [hunk 17]
```diff
@@ -166,14 +203,24 @@ SaleResponse response = POS.Instance.Sale(ammount, ticket);
...
 
-El resultado de la venta se entrega en la forma de un objeto `SaleResponse` o un `char*` en el caso de la librería C. Si ocurre algún error al ejecutar la acción en el POS se lanzará una excepción del tipo `TransbankSaleException`.
+```Java
+import cl.transbank.pos.POS;
+//...
+SaleResponse saleResponse = POS.getInstance().sale(amount, ticket);
+```
+
+
+
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```

File: documentacion/posintegrado/README.md [hunk 18]
```diff
@@ -12,20 +12,30 @@ También puede probar las DLLs que se adjuntan en el [último release](https://g
...
 Recuerda que necesitas tener instalados los drivers correspondientes a tu tarjeta de
 puerto serial o adaptador USB Serial.
 
 <aside class="notice">
-La comunicación con el POS Integrado se realiza mediante puerto serial RS232 y
-que tú eres el responsable de instalar el driver correcto para tu tarjeta o adaptador serial.
+La comunicación con el POS Integrado se realiza mediante puerto serial RS232 y tú eres el responsable de instalar el driver correcto para tu tarjeta o adaptador serial.
 </aside>
 
+
```
Comment (afiebig) [inline documentacion/posintegrado/README.md]:
```suggestion
```

File: documentacion/posintegrado/README.md [hunk 19]
```diff
@@ -12,18 +12,27 @@ También puede probar las DLLs que se adjuntan en el [último release](https://g
 
 Esta librería y sus dependencias son requisitos para utilizar el SDK.
 
-Por el momento, el SDK está disponible para [.Net](https://github.com/TransbankDevelopers/transbank-pos-sdk-dotnet) y lo puedes encontrar en [NuGet.org](https://www.nuget.org/packages/TransbankPosSDK/) para instalarlo puedes utilizar por ejemplo el package manager de VisualStudio.
+Por el momento, el SDK está disponible para [.NET](https://github.com/TransbankDevelopers/transbank-pos-sdk-dotnet) y [Java](https://github.com/TransbankDevelopers/transbank-pos-sdk-java). Para .NET lo puedes encontrar en [NuGet.org](https://www.nuget.org/packages/TransbankPosSDK/) para instalarlo puedes utilizar por ejemplo el package manager de VisualStudio.
```
Comment (gonzunigad) [inline documentacion/posintegrado/README.md]: is there a maven URL that we can share in here?

File: documentacion/posintegrado/README.md [hunk 20]
```diff
@@ -12,18 +12,27 @@ También puede probar las DLLs que se adjuntan en el [último release](https://g
...
 
 ```bash
 PM> Install-Package TransbankPosSDK
 ```
 
+Para Java se puede incluir el paquete por [Maven.](https://search.maven.org/artifact/com.github.transbankdevelopers/transbank-pos-sdk-java/)
+
+```pom.xml
```
Comment (gonzunigad) [inline documentacion/posintegrado/README.md]:
```suggestion
```xml
```

File: documentacion/posintegrado/README.md [hunk 21]
```diff
@@ -12,18 +12,27 @@ También puede probar las DLLs que se adjuntan en el [último release](https://g
...
+		</dependency>
+```
+
 Recuerda que necesitas tener instalados los drivers correspondientes a tu tarjeta de
 puerto serial o adaptador USB Serial.
 
 <aside class="notice">
-La comunicación con el POS Integrado se realiza mediante puerto serial RS232 y
-que tú eres el responsable de instalar el driver correcto para tu tarjeta o adaptador serial.
+La comunicación con el POS Integrado se realiza mediante puerto serial RS232 y tú eres el responsable de instalar el driver correcto para tu tarjeta o adaptador serial.
```
Comment (gonzunigad) [inline documentacion/posintegrado/README.md]:
```suggestion
La comunicación con el POS Integrado se realiza mediante puerto serial RS232 y tú eres el/la responsable de instalar el driver correcto para tu tarjeta o adaptador serial.
```

File: documentacion/posintegrado/README.md [hunk 22]
```diff
@@ -92,11 +107,18 @@ List<string> ports = Serial.ListPorts();
...
+import cl.transbank.pos.POS;
+
+POS pos = POS.getInstance();
+List<String> ports = pos.listPorts();
+```
+
 ### Abrir un puerto Serial
 
-Para abrir un puerto serial y comunicarte con el POS Integrado, necesitarás el nombre del puerto (El cual puedes identificar usando [la función mencionada en el apartado anterior](/documentacion/posintegrado#listar-puertos-disponibles)). También necesitarás el baudrate al cual esta configurado el puerto serial del POS Integrado (Por defecto es 115200), y puedes obtener los distintos valores desde la clase `TbkBaudrates` del paquete `Transbank.POS.Utils`.
+Para abrir un puerto serial y comunicarte con el POS Integrado, necesitarás el nombre del puerto (El cual puedes identificar usando [la función mencionada en el apartado anterior](/documentacion/posintegrado#listar-puertos-disponibles)). También necesitarás el baudrate al cual esta configurado el puerto serial del POS Integrado (Por defecto es 115200), y puedes obtener los distintos valores desde la clase `Transbank.POS.Utils.TbkBaudrates` en .NET y la clase`cl.transbank.pos.utils.TbkBaudRate` en Java.
```
Comment (gonzunigad) [inline documentacion/posintegrado/README.md]:
```suggestion
Para abrir un puerto serial y comunicarte con el POS Integrado, necesitarás el nombre del puerto (el cual puedes identificar usando [la función mencionada en el apartado anterior](/documentacion/posintegrado#listar-puertos-disponibles)). También necesitarás el _baudrate_ al cual esta configurado el puerto serial del POS Integrado (por defecto es 115200), y puedes obtener los distintos valores desde la clase `Transbank.POS.Utils.TbkBaudrates` en .NET y la clase `cl.transbank.pos.utils.TbkBaudRate` en Java.
```

File: documentacion/posintegrado/README.md [hunk 23]
```diff
@@ -92,11 +107,18 @@ List<string> ports = Serial.ListPorts();
...
 
-Si el puerto no puede ser abierto, se lanzará una exception `TransbankException`.
+Si el puerto no puede ser abierto, se lanzará una exception `TransbankException` en .NET y Java.
```
Comment (gonzunigad) [inline documentacion/posintegrado/README.md]:
```suggestion
Si el puerto no puede ser abierto, se lanzará una excepción `TransbankException` en .NET y Java.
```

File: documentacion/posintegrado/README.md (hunk 19 above)
Comment (gonzunigad) [reply documentacion/posintegrado/README.md]: ?

Comment (afiebig) [reply documentacion/posintegrado/README.md]: This URL goes to Github. In the next section is the URL to Maven (which doesn't exist right now)

**Pull request merged**